schema/va-spec/base/va-spec-source.yaml

```diff
@@ -1035,7 +1027,7 @@ $defs:
     maturity: draft
     type: object
     description: >-
-      A proposition describing the role of a variant in causing an inherited condition.
+      A proposition describing the role of a variant in causing a heritable condition.
```

Comment:
Note the slight difference between "inherited" and "heritable" here. _de novo_ variants, for example, are heritable (and interpretable by ACMG guidelines) but not inherited.